doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -192,8 +191,7 @@ Here's a summary of what icons and statuses you can expect:
   - Open Tools / Options
   - Search for **.NET / C++ Hot Reload**
   - Ensure that all three checkboxes are checked (_**Enable hot reload when debugging**_, _**Enable Hot Reload without debugging**_ and _**Apply Hot Reload on File Save**_)
-- Hot Reload for WebAssembly is not supported when using the debugger. Start your app using `Ctrl+F5`.
-- The output window in VS has an output named `Uno Platform` in its drop-down. Diagnostics messages from the VS integration appear there. Changing the MSBuild build output verbosity in `Tools/Options/Projects and Solutions/Build And Run` controls the log level.
+- The output window in VS has an output namegd `Uno Platform` in its drop-down. Diagnostics messages from the VS integration appear there. Changing the MSBuild build output verbosity in `Tools/Options/Projects and Solutions/Build And Run` controls the log level.
```

Comment:
```suggestion
- The output window in VS has an output named `Uno Platform` in its drop-down menu. Diagnostic messages from the VS integration appear there. Changing the MSBuild build output verbosity in Tools/Options/Projects and Solutions/Build And Run controls the log level.
```

Reply:
Also @jeromelaban would be nice to specify which level of verbosity we should have at minimum to see logs in the "Uno Platform" output (+ VS restart needed when changing the verbosity level?)